chat-commands.js

```diff
@@ -3300,6 +3300,7 @@ exports.commands = {
 		}
 		if (!target) return this.errorReply("Provide a valid format.");
 		let originalFormat = Tools.getFormat(target);
+		// Client handles sending a default format if none is selected.
```

Comment:
I see no reason for not having both, as this covers someone using /vtm manually, and i dont see a good reason to open a PR to revert a PR.